multinet/util.py

```diff
@@ -4,14 +4,39 @@
 
 from flask import Response
 
-from typing import Sequence, Any, Generator
+from typing import Sequence, Any, Generator, Dict, Set
 
 from . import db
 from .errors import DatabaseNotLive, DecodeFailed
+from .types import EdgeTableProperties
 
 TEST_DATA_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "../test/data"))
 
 
+def get_edge_table_properties(workspace: str, edge_table: str) -> EdgeTableProperties:
```

Comment:
In general throughout the function, a few brief comments explaining the large scale structure of what's happening would be helpful.

Reply:
And now that I'm thinking about it, perhaps unit tests for this would be a good idea too.

We can add those in a future PR, but please open an issue around it.